main/editor.js

```diff
@@ -17,11 +17,26 @@ const url = isDev ? devPath : prodPath;
 
 let editor = null;
 
-const openEditorWindow = () => {
+const openEditorWindow = ({alwaysOnTop = false} = {}) => {
```

Comment:
Is this just for testing? Or do we have a use-case for it?

Reply:
It's just for when using my test/editor.js script to launch it with alwaysOnTop, I find it easier to develop with it always on top since then i can have it over my editor and see it all the time 👍